pvmismatch/contrib/gen_coeffs/__init__.py

```diff
@@ -11,25 +11,37 @@
 # IEC 61853 test matrix
 TC_C = [15.0, 25.0, 50.0, 75.0]
 IRR_W_M2 = [100.0, 200.0, 400.0, 600.0, 800.0, 1000.0, 1100.0]
-TEST_MAT = np.meshgrid(TC_C, IRR_W_M2)
+TEST_MAT = np.meshgrid(TC_C, IRR_W_M2)  #: IEC61853 test matrix
 
 def gen_iec_61853_from_sapm(pvmodule):
     """
     Generate an IEC 61853 test from Sandia Array Performance Model (sapm).
 
-    :param pvmodule: PV module to be tested
-    :type pvmodule: dict
+    :param dict pvmodule: PV module to be tested
```

Comment:
Thanks for adding docstrings everywhere! 
Our future users will thanks us plenty for that :)